ros2/ros2/Dockerfile

```diff
@@ -51,13 +55,16 @@ RUN apt-get update && apt-get install -q -y \
     && rm -rf /var/lib/apt/lists/*
 
 # setup environment
-RUN locale-gen en_US.UTF-8
+RUN apt-get update && apt-get install -y \
```

Comment:
It's not this PR's change.
I used https://github.com/osrf/docker_images/blob/master/docker/Dockerfile to run `ros2/create_dockerfiles.py` and the auto-generated Dockerfile was changed them.
But no changes to `ros/kinetic/*/Dockerfile` by `ros/create_dockerfiles.py`.